ncm-metaconfig/src/main/metaconfig/udev/blockdev.tt

```diff
@@ -0,0 +1,8 @@
+[%- FOREACH dv IN scsi_run -%]
+SUBSYSTEM=="block", SUBSYSTEMS=="scsi", PROGRAM=="/lib/udev/scsi_id -g -u -d /dev/%k", RUN+="[% dv  %]"
```

Comment:
it would be really great if you could generate the `/sys/block/` echo commands like https://github.com/finley/GPFS-Goodies/blob/master/sbin/tune_block_device_settings#L431 (i'm assuming that's what your magic script does 😄 

also, this is only useful if after the changes https://github.com/finley/GPFS-Goodies/blob/master/sbin/tune_block_device_settings#L568 is applied. this should be mentioned somewhere